README.md

```diff
@@ -42,6 +43,9 @@ make run/docker/up
 make run/log_gen
 ```
 
+If you have an AWS S3 bucket, you can store loki's chunks in it with
+make run/loki/s3
```

Comment:
nit: wrap in code-block formatting and explain the user that he has to set his s3 credentials first

Reply:
Done. Thanks for catching this.